data/NV/incentives.json

```diff
@@ -135,37 +141,19 @@
     ],
     "item": "weatherization",
     "program": "nv_wellsRuralElectric_saveEnergy,SaveMoney",
-    "amount": {
```

Comment:
this diff is a little confusing, but I combined NV-7 and NV-8 (into NV-7) since they're both variations on the same service. same for NV-9 and NV-10 below